README.md

```diff
@@ -20,47 +22,44 @@ An SDK to reduce your development effort when building side-by-side extension ap
 - [Links](#links)
 - [License](#license)
 
-## Packages
+### Packages
```

Comment:
Hmm, I would prefer this and the following sections to be one level up (## instead of ###), they are not subsections of the TOC, are they?
Why did you move them down?